winlogbeat/eventlog/wineventlog.go

```diff
@@ -600,3 +628,112 @@ func (c *winMetaCache) winMeta(provider string) *winevent.WinMeta {
 	c.cache[provider] = winMetaCacheEntry{expire: time.Now().Add(c.ttl), WinMeta: &s.WinMeta}
 	return &s.WinMeta
 }
+
+// incrementMetric increments a value in the specified expvar.Map. The key
```

Comment:
Curious why this is moved?

Reply:
The linux linter failed because these items were "unused" in that arch, so I had to move them to a windows build-only file.

Reply:
OK. Thanks.